### ISSUE TITLE
Show CSV details in catalog / preview window

### ISSUE THREAD

**Issue by stevage** [TerriaJS/terriajs#1369]
We don't show much information about the CSV file we've loaded. Helpful information would include:
- whether region mapped or lat-lon
- what region column is being used, and how it's being interpreted
- what other special columns are identified
- how many rows
- how many columns

**Comment by meh9** [comment]
Yes, this would be really useful to help figure out just why things look the way they look.
